### ISSUE TITLE
Add proxy admin dashboard page

### ISSUE THREAD

**Issue by carolynvs** [gomods/athens#360]
It would be useful if the proxy hosted a dashboard with stats:

* disk usage
* requests per package
* other dashboardy info that we have readily available

**Comment by michalpristas** [comment]
can we make this part of some kind of healthcheck endpoint?

**Comment by arschles** [comment]
I think we should have both, but would you be ok if we put the healthcheck
endpoint into another issue?

**Comment by michalpristas** [comment]
yes absolutely, i just wanted to point out that this kind of information i would like to have in a healthcheck as well.

**Comment by arschles** [comment]
all the ❤️ s

**Comment by robjloranger** [comment]
see #446 for metric details

**Comment by arschles** [comment]
@robjloranger do you still think we should do this?

**Comment by robjloranger** [comment]
personally I do like dashboards. Would it be public or do we need to add some user management stuff, or at least a dashboard password?

**Comment by arschles** [comment]
At the moment, there is no password protection on any of the other Athens endpoints, so I think this can be passwordless.

To do a dashboard with things like @carolynvs suggested in above:

>disk usage
>requests per package

Athens would need to start tracking these things. I would like to keep the Athens codebase, or at least the binary built from `cmd/proxy/main.go`, from growing significantly, so I would propose we do those things and serve the dashboard from a different process. Ideally a different repo. What do you think?

**Comment by matt0x6F** [comment]
I renamed this issue because what it describes is information pertaining to administering Athens. I'm opening a separate issue for a sort of "splash" page at the root URL that informs users on how to connect to a given instance that will be much more lightweight than what's described here.
